_posts/2024-04-24-ASUworkshop.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Save the Date - in-person workshop at Arizona State University
```

Comment:
I don't know how it renders, but a potential enhancement might be to add the date in the title for quick reading